skbio/io/format/genbank.py

```diff
@@ -429,7 +431,6 @@ def _parse_single_genbank(chunks):
         elif header == 'FEATURES':
             # merge the two dictionaries together. Note this is only
             # available in Python-3.5
-
             feature_metadata = merge_dicts(parsed, feature_metadata)
         else:
             metadata[header] = parsed
```

Comment:
before returning, add a line to store `locus_name` as `metadata["id"]`? currently the `id` is empty, which is not good.